formulas/wick-base/README.md

```diff
@@ -557,6 +620,28 @@ Examples
 Returns nothing.
 
 
+`wickServiceIsEnabled()`
+------------------------
+
+Internal: Figures out if a service exists and is enabled. Enabled means it will start on boot.
+
+* $1 - Service name.
+
+Examples:      wickServiceIsEnabled mysql
```

Comment:
Example formatting is messed up and it doesn't really show how it could be used.

```
Examples

    if wickServiceIsEnabled mysql; then
        echo "MySQL is enabled"
    fi
```